02-peer-protocol.md

```diff
@@ -94,6 +94,7 @@ desire to set up a new channel.
    * [`4`:`feerate_per_kw`]
    * [`2`:`to_self_delay`]
    * [`2`:`max_accepted_htlcs`]
+   * [`1`:`announce_channel`]
```

Comment:
Yes, lets switch:

* announce_channel -> flags
* "`true` (`0x01`)" to "least significant bit is set"
* "`false` (`0x00`)" to "least significant bit is clear".